keycloak/themes/haven/account/template.ftl

```diff
@@ -1,19 +1,22 @@
 <#macro mainLayout active bodyClass>
 <!doctype html>
 <html>
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-    <meta name="robots" content="noindex, nofollow">
+  <head>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0, shrink-to-fit=no">
+      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+      <meta name="robots" content="noindex, nofollow">
 
-    <title>Account Management</title>
-    <link rel="icon" href="${url.resourcesPath}/img/favicon-base.png" />
-    <#if properties.styles?has_content>
-        <#list properties.styles?split(' ') as style>
-            <link href="${url.resourcesPath}/${style}" rel="stylesheet" />
-        </#list>
-    </#if>
+      <title>Account Management</title>
+      <link rel="icon" href="${url.resourcesPath}/img/favicon-base.png" />
+      <#if properties.styles?has_content>
+          <#list properties.styles?split(' ') as style>
+              <link href="${url.resourcesPath}/${style}" rel="stylesheet" />
+          </#list>
+      </#if>
+      <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.1/js/bootstrap.min.js"></script>

```

Comment:
I know the main webUI is doing this too, I filed HAV-62 about switching to load all javascript/css from our own site rather than from a CDN.